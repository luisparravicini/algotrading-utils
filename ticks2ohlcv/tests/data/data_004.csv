1449999960,120.00000,0.40000000
1449999960,121.00000,0.40000000
1449999961,120.00000,0.40000000
1450000020,120.00000,0.40000000
