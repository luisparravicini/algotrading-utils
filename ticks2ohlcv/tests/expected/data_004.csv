date,open,high,low,close,volume
2015-12-13 06:46:00,120.0000,121.0000,120.0000,120.0000,0.0000
2015-12-13 06:47:00,120.0000,120.0000,120.0000,120.0000,0.0000
